nntrainer/src/bn_layer.cpp

```diff
@@ -87,69 +87,71 @@ int BatchNormalizationLayer::setProperty(std::vector<std::string> values) {
 
 Tensor BatchNormalizationLayer::forwarding(Tensor in, int &status) {
   Tensor temp;
-  assert(dim.batch() > 0);
-  hidden = in;
+  this->input = in;
```

Comment:
Lets add a comment here at the top that this is for `TRAIN` mode, and `EVAL` mode is a TODO

Reply:
done.